Userland/Libraries/LibAudio/GenericTypes.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022, Lucas Chollet <lucas.chollet@free.fr>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/String.h>
+#include <AK/Vector.h>
+
+namespace Audio {
+
+enum class FlacPictureType : u32;
+
+struct PictureData {
```

Comment:
Please clarify that this data type can both be used for APIC in ID3v2 and PICTURE in FLAC.